device-wallet.js

```
@@ -28,26 +28,31 @@ const setAutoPressButton = function(value, def, circular) {
     } else if ( Array.isArray(def) ) {
       sequence = [].concat(def);
     } else {
-      return;
+      return 'Only string or Array allowed as parameter';
```

`throw` instead of `return` .